Plugin/src/SofaPython3/PythonEnvironment.h

```diff
@@ -120,7 +120,7 @@ class SOFAPYTHON3_API PythonEnvironment
     struct SceneLoaderListerner : public SceneLoader::Listener
     {
         /// possibly unload python modules to force importing their eventual modifications
-        virtual void rightBeforeLoadingScene();
+        virtual void rightBeforeLoadingScene() override;
```

Comment:
```suggestion
        virtual void rightBeforeLoadingScene(SceneLoader* sceneLoader) override;
```

Reply:
This also implies that `sceneLoader` is unused in the implementation of this function in the cpp file.

Reply:
This one was not just a warning. It is a bug introduced by https://github.com/sofa-framework/sofa/pull/3799 because the `override` keyword was missing in the first place.